### ISSUE TITLE
Support for Champion/Warrior Medals?

### ISSUE THREAD

**Issue by jsj1027** [Phlarx/tm-ultimate-medals#49]
I know these plugins have this built in, but it would be nice to have it all in 1 UI.

**Comment by Catalistique** [comment]
I made exactly that a few weeks ago so, seeing this, I decided to publicly release it on Openplanet as [Ultimate Medal++](https://openplanet.dev/plugin/ultimate_medal_plusplus)

**Comment by zai-tm** [comment]
> I made exactly that a few weeks ago so, seeing this, I decided to publicly release it on Openplanet as [Ultimate Medal++](https://openplanet.dev/plugin/ultimate_medal_plusplus)

Why didn't you just make this a PR for this plugin? It doesn't need to be in it's own plugin.

**Comment by XertroV** [comment]
#50 fixes this if it ever gets merged

**Comment by Catalistique** [comment]
> Why didn't you just make this a PR for this plugin? It doesn't need to be in it's own plugin.

Yeah, you're not the only one who told me this and honestly you're right. However, my code was not the greatest and I had never contributed to an open-source project.

Since XertroV and Nsgr did it too, I don't know if it fixes anything, but I rewrote the plugin and also made a pull request. Sorry again to everyone, made a lapse of judgement. Btw, if there's any way to unrelease a plugin, please tell me and I'll do it as soon as the merge is made.

[The pull request I made](https://github.com/Phlarx/tm-ultimate-medals/pull/51)

**Comment by Phlarx** [comment]
Both myself and Miss are happy to keep this plugin as it currently is, displaying only the information provided by the game, and letting Ultimate Medals++ and Ultimate Medals Extended take on the addition of external medals such as champion and warrior.

If any of that changes, we may re-visit this.
